R/fsi_functions.R

```diff
@@ -47,6 +47,33 @@ fsi_create <- function(name, and_method = "min",
   fsi
 }
 
+
+
+#' @title fsi_add_fsa
+#' 
+#' @description fsi_add_fsa adds a fuzzy spatial antecedent to the FSI model
+#' 
+#' @usage 
+#' 
+#' fsi_add_fsa(fsi, lvar, tbl)
+#' 
+#' @param fsi The FSI model instantiated with the `fsi_create` function.
+#' @param lvar A character value that represents a linguistic variable of the antecedent.
+#' @param tbl  A tibble with spatial plateau objects annotated with linguistic values of the linguistic variable specified by the above `lvar` parameter.
+#' 
+#' @details 
+#' 
+#' The fuzzy spatial antecedent added by the `fsi_add_fsa` function is composed of a linguistic variable and its corresponding `pgeom objects` annotated by linguistic values. 
+#' The format of the `tbl` parameter is the same as the output of the function `spa_creator`, allowing the user to directly provides plateau region objects as input when designing FSI models.
+#' 
+#' @returns 
+#' 
+#' An FSI model populated with a fuzzy spatial antecedent.
+#' 
+#' @example 
+#' 
+#' fsi <- fsi_add_fsa(fsi, "accommodation price", accom_price_layer)
```

Comment:
Please, provide a more complete example. That is, the example should include the creation of the FSI model, the creation of a tibble for the parameter `tbl` (i.e., resulting from the execution of the function `spa_create`), and then its inclusion to the FSI model.